tailwind.config.ts

```
@@ -2,13 +2,64 @@ import { type Config } from "tailwindcss";
+  			chart: {
+  				'1': 'hsl(var(--chart-1))',
+  				'2': 'hsl(var(--chart-2))',
+  				'3': 'hsl(var(--chart-3))',
+  				'4': 'hsl(var(--chart-4))',
+  				'5': 'hsl(var(--chart-5))'
+  			}
+  		}
+  	}
   },
-  plugins: [],
+  plugins: [require("tailwindcss-animate")],
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)